apps/pg_subscriber/lib/pg_subscriptor/handler.ex

```diff
@@ -134,6 +140,31 @@ defmodule PgSubscriber.Handler do
     Logger.info(tuple_data: tuple_data)
   end
 
+  defp handle_delete!(body) do
+    Logger.info("Got DELETE msg")
+    <<_relation_oid::32, tuple_type::8, rest::binary>> = body
+    {:ok, tuple_data, <<>>} = TupleData.get_tuple_data(rest)
+
+    Logger.info(tuple_type: <<tuple_type>>)
+    Logger.info(tuple_data: tuple_data)
+  end
+
+  defp handle_truncate!(body) do
+    Logger.info("Got TRUNCATE msg")
+    <<relations_num::32, _options::8, rest::binary-size(4 * relations_num)>> = body
```

Comment:
Just to think about. Is size necessary? You can just take the rest and let it crash if something is wrong. If we were handling errors, then maybe, but you do not check whether the message is big enough for `relations_num::32` neither for example...

Reply:
With exact check for size I wanted to shows that `TRUNCATE` is one of the messages that can be parsed right away. It seems a bit better to fail right away and not during parsing of relations.